charts/prometheus-pushgateway/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.4.2"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 1.18.3
+version: 1.18.4
```

new feature should raise minor version

```suggestion
version: 1.19.0
```

---

done